sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-b2c-oidc/README.md

```diff
@@ -47,6 +47,7 @@ azure:
       client-secret: ${your-client-secret}
       reply-url: ${your-reply-url} # should be absolute url.
       logout-success-url: ${your-logout-success-url}
+      user-name-attribute-name: name
```

Comment:
![image](https://user-images.githubusercontent.com/4465723/104869005-d4275c00-597f-11eb-99bd-c5dd3c4bfcc6.png)
I think we need to change the task link to issue url